src/app.js

```diff
@@ -15,6 +15,8 @@ const urlJoin = require('url-join');
 
 const config = require('./config');
 const conversationLogger = require('./conversationLogger');
+const dispatcher = require('./dispatcher');
```

Comment:
@crccheck the idea is to have `dispatcher` be the event bus between various separations of concerns. This is a first step toward that. The next is pulling the Express code into its own code and the session code out into its own module. This the first step toward that.